doc/source/notebooks/manual/advanced/ordinal_regression.ipynb

```diff
@@ -7,7 +7,212 @@
+      "text/plain": [
+       "<Figure size 432x288 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "#make a one dimensional ordinal regression problem\n",
+    "np.random.seed(1)\n",
+    "num_data = 20\n",
+    "X = np.random.rand(num_data, 1)\n",
+    "\n",
+    "# First, generate values of a latent GP\n",
+    "K = np.exp(-0.5*np.square(X - X.T)/0.01) + np.eye(num_data)*1e-6\n",
```

Comment:
I don't think this reads as easy as intended.
I'd propose something in the line of
```
kern = gpflow.kernels.RBF(1, lengthscales=0.1)
K = kern.compute_K_symm(X)
f = np.random.multivariate_normal(mean=np.zeros(num_data), cov=K).reshape(-1, 1)
```